test/SparkLendMainnetIntegration.t.sol

```diff
@@ -32,43 +32,29 @@ contract SparkLendMainnetIntegrationTest is Test {
     address POT                     = 0x197E90f9FAD81970bA7976f33CbD77088E5D7cf7;
     address ADMIN                   = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;  // SubDAO Proxy
 
-    address DAI  = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
-    address ETH  = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
-    address USDC = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
-    address USDT = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+    address DAI    = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address ETH    = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address USDC   = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address USDT   = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+    address STETH  = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84;
+    address WSTETH = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+    address RETH   = 0xae78736Cd615f374D3085123A210448E74Fc6393;
 
     address ETH_IRM       = 0xeCe550fB709C85CE9FC999A033447Ee2DF3ce55c;
     address USDC_ORACLE   = 0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6;
     address USDT_ORACLE   = 0x3E7d1eAB13ad0104d2750B8863b489D65364e32D;
     address USDC_USDT_IRM = 0xbc8A68B0ab0617D7c90d15bb1601B25d795Dc4c8;  // Note: This is the same for both because the parameters are the same
+    address ETHUSD_ORACLE = 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419;
+    address RETH_ORACLE   = 0x05225Cd708bCa9253789C1374e4337a019e99D56;
+    address WSTETH_ORACLE = 0x8B6851156023f4f5A66F68BEA80851c3D905Ac93;
 
     IAaveOracle            aaveOracle            = IAaveOracle(AAVE_ORACLE);
     IPoolAddressesProvider poolAddressesProvider = IPoolAddressesProvider(POOL_ADDRESSES_PROVIDER);
     IPool                  pool                  = IPool(POOL);
     IPoolConfigurator      configurator          = IPoolConfigurator(POOL_CONFIGURATOR);
 
     function setUp() public {
-        vm.createSelectFork(getChain("mainnet").rpcUrl, 18784436);  // Dec 14, 2023
-    }
-
-    function test_dai_market_oracle() public {
```

Comment:
Same here an keep the test and set the block within it

Reply:
readded